oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/search/ActiveFiltersElement.jsx

```diff
@@ -7,16 +7,24 @@ import { withState } from "react-searchkit";
 import { ClearFiltersButton } from "@js/oarepo_ui";
 import { useActiveSearchFilters } from "./hooks";
 
-const getLabel = (filter, aggregations) => {
+const getLabel = (filter, aggregations, additionalFilterLabels) => {
   const aggName = filter[0];
   let value = filter[1];
+  // keep the original methodo of getting labels for backwards compatibility just in case
+  // to not break existing applications
   const label =
     aggregations[aggName]?.buckets?.find((b) => b.key === value)?.label ||
+    additionalFilterLabels[aggName]?.value_labels?.find((b) => b.key === value)
```

Comment:
Why not to store it also under `buckets`? Then this code could be extracted into `getValueLabel` helper to make things more readable